apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -214,7 +214,7 @@ type KafkaConfig struct {
 // TODO: Other potential modes: RDS and Operator (e.g. CrunchyDB)
 
 // DatabaseMode details the mode of operation of the Clowder Database Provider
-// +kubebuilder:validation:Enum=app-interface;local;none
+// +kubebuilder:validation:Enum=single;app-interface;local;none
```

Comment:
should we change the name from single to shared now that we could have more than one?

Reply:
yeh